warehouse/models/staging/ntd_annual_reporting/_stg_ntd_annual_reporting.yml

```diff
@@ -174,6 +174,9 @@ x-common-fields:
     description: '{{ doc("ntd_execution_timestamp") }}'
 
 models:
+  - name: stg_ntd__2022_agency_information
+  - name: stg_ntd__2023_agency_information
+  - name: stg_ntd__2023_contractual_relationships
```

Comment:
Looks like it is erroring because these two tables as duplicated on this file. Just need to delete them from lines 501, 502